frontend/lib/ifixit-api/devices.ts

```diff
@@ -5,6 +5,16 @@ import { z } from 'zod';
 
 export type DeviceWiki = Record<string, any>;
 
+export type DeviceWikiApiResponse = {
```

Comment:
Naming: Doesn't this type represent a `ProductApiResponse`, not a device wiki api response? I also feel like this type and the fetcher don't belong in the `devices.ts` file, but maybe a new `product.ts` file

Reply:
this got merged so fast after qa that i didn't get aroudn to addressing these.I guess ill staple these changes on to the inevitable compatibility notes PR